gdeploy_config/volume_usmqe_gama_disperse_4_plus_2x2.create.conf

```diff
@@ -0,0 +1,24 @@
+#   plus 1 block device as arbiter named (/dev/vdd)
```

Comment:
Could you add slightly more descriptive comment here (for me it seems, like there is something missing)?
And maybe it might be worth to update comments for all configs with some description/link what it configure, not only state the requirements.

Reply:
It might be helpful to have link to some description of the particular volume type, but if you think it's ok as it is now, I'm ok with that too.

Reply:
I think this is good idea, I've added links to US docs.